vulture/core.py

```diff
@@ -244,16 +253,18 @@ def report(self):
         Print ordered list of Item objects to stdout.
         """
         for item in self.get_unused_code():
-            if self.sort_by_size:
-                line_format = 'line' if item.size == 1 else 'lines'
-                size_report = ' ({0:d} {1})'.format(item.size, line_format)
-            else:
-                size_report = ''
-
-            print("{0}:{1:d}: {2}{3}".format(
-                utils.format_path(item.filename), item.lineno, item.message,
-                size_report))
-            self.found_dead_code_or_error = True
+            item.confidence >= self.min_confidence
+            if item.confidence >= self.min_confidence:
+                if self.sort_by_size:
+                    line_format = 'line' if item.size == 1 else 'lines'
+                    size_report = ' ({0:d} {1})'.format(item.size, line_format)
+                else:
+                    size_report = ''
+
+                print("{0}:{1:d}: {2}{3} (sureness: {4})".format(
+                    utils.format_path(item.filename), item.lineno,
+                    item.message, size_report, item.confidence))
+                self.found_dead_code_or_error = True
```

Comment:
I'd like to use the format

    foo.py:2: unused function 'bar' (60% confidence, 12 lines)
    foo.py:2: unused function 'bar' (60% confidence)